exercise_2/openweather_api_client.py

```diff
@@ -0,0 +1,45 @@
+import requests
+import collections
+
+
+WeatherForecast = collections.namedtuple(
+    "WeatherForecast", ("city", "country", "current_weather", "temperature", "pressure")
+    )
+
+
+class BasicWeatherApiClient:
+    """Basic abstraction for services to handle serializing data from service
```

Comment:
Oki, docstring jest fajny i sensowny i to jest coś czego bym oczekiwał, jednak nie ma to się jak do aktualnej implementacji.